portfolio/src/main/java/com/google/sps/servlets/HelloWorldServlet.java

```diff
@@ -1,18 +1,50 @@
 package com.google.sps.servlets;
 
 import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Random;
+
+import com.google.gson.Gson;
+
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
 /** Handles requests sent to the /hello URL. Try running a server and navigating to /hello! */
 @WebServlet("/hello")
-public class HelloWorldServlet extends HttpServlet {
+public final class HelloWorldServlet extends HttpServlet {
+
+  private static final ArrayList<String> MOTIVATIONS = new ArrayList<String>(Arrays.asList(
+                "If at first you do not succeed, try again!", "Yes you can!", "If it was easy, everyone would do it!"));
 
```

Comment:
Format this so that each line is shorter
  MOTIVATIONS =
       new ArrayList<> (
         Arrays.asList(
          "If at ..",
          "Yes you.."));